modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc

```diff
@@ -10,21 +10,95 @@ image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", ti
+|P0 | Critical | 7 days
+|P1 | High | 30 days
+|===
+
+
+== Monthly Date Patch Updates 
+
+Once per month, MuleSoft releases a _date patch_ update, 
+which includes backward-compatible bug fixes to address any issues discovered in Mule runtime engine.
+The monthly date patch update might include security patches if the scheduled date is within the security SLA.
+
+The format of the version number for these updates is:
+
+_Major_._Minor_._Patch_ - _Date_ 
+
+Here's an example version number, including the patch date:
+
+`4.2.2 - 05-04-2020`
+
+The monthly patches only update applications to the latest date patch for the patch version.
+They do not change the patch version number.
+
+
+=== Update Schedule 
+
+The schedule for date patch updates is:
+
+* In the first full week of the month, MuleSoft releases the date patch.
+* In the third week of the month, MuleSoft automatically applies the date patch that was released in the first week of the month.
++
+Applications in environments other than Production are updated during the week.
+Production applications are updated on the weekend.
++
+When an automatic update occurs, the audit log for your app contains an entry by user `Anypoint Staff`,
+which shows the date and time an update occurred and whether it was successful.
++
+If the update is successful, no action is required.
+See <<Zero-Downtime Restart Limitations>>.
+
+
+
+If your application does not support zero-downtime updates or requires a manual update,
+you can apply the date patch released in the first week of the month in week 1 or week 2. 
+See <<Update the Runtime Version Manually>>.
+
+To update multiple applications, use the CloudHub API.
+See <<Update Multiple Apps to the Latest Runtime Release>>.
+
+=== Updates Available 
+
+When a new runtime version is available:
+
+* The *Applications* page displays a banner and the *Update Available* tab lists the apps that require updates:
++
 .The screenshot shows (*1*) the banner and (*2*) the *Update Available* tab on the *Applications* page.
 image::cloudhub-app-update-req-page.png[Applications page with banner and Update Available tab]
-
++
 If no apps require updates, no banner appears and the *Update Available* tab is empty.
 
-== Update the App Manually 
+*  *Update available* and the new date patch appear in the *Runtime version* menu in the *Settings* page for the application:
++
+.The screenshot shows (*1*) the date patch version update and (*2*) *Update available* in the *Runtime version* menu.
+image::cloudhub-app-update-versions.png[Date patch version update and Update available in the Runtime version menu.]
 
-The *Update Available* tab indicates that one or more apps require an update. You can schedule the update to occur at your convenience.
+
+MuleSoft applies the monthly updates automatically, based on the published schedule. 
+If you want, you can update them manually at your convenience.
+
+== Update the Runtime Version Manually 
```

Comment:
Just checking: this is intended as a peer topic of monthly date patch updates, which has subtopics for scheduled and available updates?
(I wonder if "available" should be "unscheduled" or "periodic"?)

Reply:
No - it's meant to describe what happens when updates are available.